### ISSUE TITLE
recline_map_view does not recognize GeoJSON fileds

### ISSUE THREAD

**Issue by u10313335** [ckan/ckan#2387]
If we try to create a recline_map_view (CKAN 2.3+) for data stored in the DataStore, the GeoJSON fields can not be listed since they are "text" fields rather than "numeric" fields.

It can be fixed by adding "text" type to the ReclineMapView extension (tested in my CKAN instance):

```diff
diff --git a/ckanext/reclineview/plugin.py b/ckanext/reclineview/plugin.py
index 20e204b..cc59163 100644
--- a/ckanext/reclineview/plugin.py
+++ b/ckanext/reclineview/plugin.py
@@ -178,7 +178,7 @@ class ReclineMapView(ReclineViewBase):

     datastore_fields = []

-    datastore_field_types = ['numeric']
+    datastore_field_types = ['numeric', 'text']

     def list_map_field_types(self):
         return [t['value'] for t in self.map_field_types]
```

If the above solution is right, I will send a PR then.

**Comment by amercader** [comment]
Fix in #2537

**Comment by u10313335** [comment]
Confirmed. Bug fixed. Thank you all!
